bin/MadGraph5_aMCatNLO/cards/pp_H_TTG_012j_5f_NLO_MidDM/pp_H_TTG_012j_5f_NLO_MidDM_proc_card.dat

```diff
@@ -0,0 +1,29 @@
+#set group_subprocesses Auto
+#set ignore_six_quark_processes False
+#set low_mem_multicore_nlo_generation False
+#set complex_mass_scheme False
+#set gauge unitary
+#set loop_optimized_output True
+#set loop_color_flows False
+#set max_npoint_for_channel 0
+#set default_unset_couplings 99
+#set max_t_for_channel 99
+#set zerowidth_tchannel True
+#import model sm
+#define p = g u c d s u~ c~ d~ s~
+#define j = g u c d s u~ c~ d~ s~
+#define l+ = e+ mu+
+#define l- = e- mu-
+#define vl = ve vm vt
+#define vl~ = ve~ vm~ vt~
+#set auto_convert_model T
+
+define p = p b b~
+define j = j b b~
+
+import model model DipoleMomentMid_UFO
```

Comment:
line 24 should be put before line 21 and 22

Reply:
I found out from a post on launchpad that I needed to be using madspin and have since included the above model there. However, if I set maxjetflavor = 5 in the run card, do I need to set p = p b b~ and j = j b b~ in the proc card as well?

Reply:
yes, you need to import the model first (btw, it should "import model DipoleMomentMid_UFO"), and then define the flavor